common/run_genesis.py

```diff
@@ -23,4 +24,4 @@ def run_genesis(top: str,
     res = os.system(cmd)
     if not res == 0:
         raise RuntimeError(f"Genesis failed! cmd = {cmd}")
-    return f"genesis_verif/{top}.v"
+    return f"genesis_verif/{top}.{'sv' if system_verilog else 'v'}"
```

Comment:
We could consider making the "top filename" be the first or only file passed via `in_file_or_files`, then we wouldn't need this suffix related code (so `in_file_or_files=top.svp` would map to `top.sv` and `in_file_or_files=[main.vp, pe.svp]` would map to `main.v`)

Reply:
is the `system_verilog` flag only used for returning the correct output verilog filename? I actually think we need to re-up the genesis wrappers, for the case where multiple files are output. We ideally are keeping track of all of the verilog files in a design (in some way, even if we don't parse them).

For now this is good.

Reply:
See #67

Reply:
Yes, it's only used for returning the correct output verilog filename. I agree that the right solution is to generalize this to work with files the genesis actually produces, rather than just the top file.

We could investigate if Genesis can provide us any information on the Verilog files that are generated, but a first pass attempt might be to assume all the files generated into the `genesis_verif` directory.

Reply:
agreed, let's go ahead with this and use #67 to track this issue.